log.out

```diff
@@ -0,0 +1 @@
+[{"pulp_created": "2021-02-13T13:33:04.220502Z", "pulp_href": "/pulp/api/v3/workers/3754d4ea-b554-432d-b20e-45026ce46c12/", "name": "resource-manager", "last_heartbeat": "2021-02-13T22:16:02.245766Z"}]
```

Comment:
Those files i would not expect here. Is the cwd mechanism not working for you?

Reply:
I don't know. I re-ran the tests and it didn't appear. Will keep an eye out.